generate_outputs.py

```diff
@@ -156,6 +161,21 @@
         with open(path, "r") as f:
             messages_datasets.append(TracedDataJsonIO.import_json_to_traced_data_iterable(f))
 
+    # TODO Update this to read from config - RecoveryCSVURLs
+    recovery_datasets = []
+    if pipeline_configuration.recovery_csv_urls is None:
```

Comment:
I see no reason to include the if on the recovery_csv_urls if you're not using them.

I would keep the TODO, but drop everything except the body of the else.

Reply:
Your are right. Updated thanks!